### PR TITLE
Migrate terra menu

### PR DESCRIPTION
**Pull request by sid141288**
### Summary
Moving code for terra-menu from old repo to new repo.
screen shot 2017-03-07 at 1 42 05 pm

### CODE REVIEW

File: packages/terra-menu/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-menu",
+  "main": "lib/Menu.js",
```
Comment (Matt-Butler) [inline packages/terra-menu/package.json]: lib/Menu.scss

File: packages/terra-menu/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
```
Comment (Matt-Butler) [inline packages/terra-menu/package.json]: This project is currently only scss. We can probably remove the js dependencies.

**Comment by alex-bezek** [comment]
+1

File: packages/terra-menu/package.json (hunk 2 above)
Comment (zhongyn) [inline packages/terra-menu/package.json]: +1

File: packages/terra-menu/package.json (hunk 1 above)
Comment (sid141288) [inline packages/terra-menu/package.json]: Updated it. :)

File: packages/terra-menu/package.json (hunk 2 above)
Comment (sid141288) [inline packages/terra-menu/package.json]: Removed js dependencies. :)

File: packages/terra-menu/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "terra-menu",
+  "main": "lib/Menu.scss",
+  "private": true,
+  "version": "0.1.0",
```
Comment (bjankord) [inline packages/terra-menu/package.json]: We need to bump this to 0.2.0 to match https://github.com/cerner/terra-menu/blob/master/package.json#L3. When we release this next, we'll release the changes as `0.3.0`

**Pull request merged**